modal-go/function.go

```diff
@@ -115,16 +115,33 @@ func (f *Function) Remote(args []any, kwargs map[string]any) (any, error) {
 	functionMapResponse, err := client.FunctionMap(f.ctx, pb.FunctionMapRequest_builder{
 		FunctionId:                 f.FunctionId,
 		FunctionCallType:           pb.FunctionCallType_FUNCTION_CALL_TYPE_UNARY,
-		FunctionCallInvocationType: pb.FunctionCallInvocationType_FUNCTION_CALL_INVOCATION_TYPE_SYNC,
+		FunctionCallInvocationType: invocationType,
 		PipelinedInputs:            functionInputs,
 	}.Build())
 	if err != nil {
 		return nil, fmt.Errorf("FunctionMap error: %v", err)
 	}
 
+	functionCallId := functionMapResponse.GetFunctionCallId()
+	return &functionCallId, nil
+}
+
+// Execute a single input into a remote Function.
+func (f *Function) Remote(args []any, kwargs map[string]any) (any, error) {
+	invocationType := pb.FunctionCallInvocationType_FUNCTION_CALL_INVOCATION_TYPE_SYNC
+	functionCallId, err := f.execFunctionCall(args, kwargs, invocationType)
+	if err != nil {
+		return nil, fmt.Errorf("FunctionMap error: %v", err)
```

Comment:
Just `return nil, err` here. Otherwise this will produce `FunctionMap error: FunctionMap error: ...` with the message displayed twice.